src/schemas/json/hammerkit.json

```
@@ -0,0 +1,349 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "additionalProperties": false,
+  "definitions": {
+    "labels": {
+      "description": "A map of label values",
+      "type": "object",
+      "additionalProperties": { "type": "string" }
+    },
```

```suggestion
    "labels": {
      "title": "label options",
      "description": "A map of label values for the current project",
      "type": "object",
      "additionalProperties": { "type": "string" }
    },
```

Mandatory fix: [unexpected `description` property, `title` property expected before it](https://github.com/SchemaStore/schemastore/blob/master/CONTRIBUTING.md#best-practices).
Mandatory fix: [type of property always should be explicitly specified: whether it's global or local (for the current project)](https://github.com/SchemaStore/schemastore/blob/master/CONTRIBUTING.md#best-practices).